.all-contributorsrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This should allow the bot to become active on the repository, allowing for people to be accredited for their contributions automatically. See: https://allcontributors.org/docs/en/bot/installation